libs/openchallenges/api-description/src/components/parameters/path/org.yaml

```diff
@@ -1,6 +1,13 @@
 name: org
 in: path
-description: The unique identifier of the organization.
+description: The id or login of the organization.
 required: true
 schema:
-  $ref: ../../schemas/OrganizationLogin.yaml
+  type: string
+  minLength: 1
+  maxLength: 25
```

Comment:
some names of orgs in current database are longer than 25 characters (i.e `nsia-kano-diagnostic-center`)- suggest using longer maxLength.

Reply:
Good catch! They should be 2-64 chars by the decision documented in this [ticket](https://github.com/Sage-Bionetworks/sage-monorepo/pull/1482).

Reply:
Here we need to support 1 as minlength to support org ID 0-9.

Reply:
This issue is fixed